openmdao/core/group.py

```diff
@@ -1435,6 +1442,9 @@ def add_subsystem(self, name, subsys, promotes=None,
             enable users to instantiate and add a subsystem at the
             same time, and get the reference back.
         """
+        if self._setup_procs_finished:
+            raise RuntimeError('Cannot call add_subsystem in the configure method')
```

Comment:
You should prepend the self.msginfo info so that the user knows which component raised the error. See the next error message below this one to see how it is done.

Reply:
Changed